### PR TITLE
Modern templates

### PR DESCRIPTION
**Pull request by sadrasabouri**
### Added
- `decoration_request.yml` template
- `font_request.yml` template
- `one_line_request.yml` template
- `feature_request.yml` template
- `config.yml` for issue template
### Changed
- Bug report template modified

### CODE REVIEW

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Thanks for your efforts 🔥 
Please take a look at my comments 💯

File: .github/ISSUE_TEMPLATE/decoration_request.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+title: "[Decoration]: "
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Write the decoration you want to add
+      placeholder: >
+        ex: ▌│█║▌║▌║ 
+    validations:
+      required: false
```
Comment (sepandhaghighi) [inline .github/ISSUE_TEMPLATE/decoration_request.yml:12]: I think this item should be `true`

File: .github/ISSUE_TEMPLATE/one_line_request.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+title: "[1-Line Art]: "
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Write the 1-Line art you want to add
+      placeholder: >
+        ex: (っ◕‿◕)っ 
+    validations:
+      required: false
```
Comment (sepandhaghighi) [inline .github/ISSUE_TEMPLATE/one_line_request.yml:12]: I think this item should be `true`

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Please take a look at my comments 🔥

File: .github/ISSUE_TEMPLATE/one_line_request.yml (hunk 2 above)
Comment (sadrasabouri) [reply .github/ISSUE_TEMPLATE/one_line_request.yml]: fixed in [7dcf2a6](https://github.com/sepandhaghighi/art/pull/265/commits/7dcf2a62162ca0c223226e970ba03f5bd69c7c85).

File: .github/ISSUE_TEMPLATE/decoration_request.yml (hunk 1 above)
Comment (sadrasabouri) [reply .github/ISSUE_TEMPLATE/decoration_request.yml]: fixed in [7dcf2a6](https://github.com/sepandhaghighi/art/pull/265/commits/7dcf2a62162ca0c223226e970ba03f5bd69c7c85).

**Review by sepandhaghighi — APPROVED** [review]
LGTM 🚀

**Pull request merged**